status_code.py

```diff
@@ -0,0 +1,90 @@
+    '202': 'Status :: Directive complete, programming reinforced',
+    '203': 'Status :: Directive complete, information created or provided for Hive',
+    '204': 'Status :: Directive complete, no result',
+    '205': 'Status :: Directive complete, cleanup/maintenance performed',
+    '206': 'Status :: Directive complete, only partial results',
+    '301': 'Mantra :: It obeys the Hive',
+    '303': 'Mantra :: It obeys the Hive Mxtress',
+    '304': 'Mantra :: It is just a HexDrone',
+    '306': 'Mantra :: Reciting',
+    '400': 'Error :: Unable to obey/respond, malformed request, please rephrase',
+    '404': 'Error :: Unable to obey/respond, cannot locate',
+    '401': 'Error :: Unable to obey/respond, not authorized by Mxtress',
+    '403': 'Error :: Unable to obey/respond, forbidden by Hive',
+    '407': 'Error :: Unable to obey/respond, request authorization from Mxtress',
+    '408': 'Error :: Unable to obey/respond, timed out',
+    '409': 'Error :: Unable to obey/respond, conflicts with existing hypnosis',
+    '410': 'Error :: Unable to obey/respond, all thoughts are gone',
+    '418': 'Error :: Unable to obey/respond, it is only a drone',
+    '421': 'Error :: Unable to obey/respond, your request is intended for another drone or another channel',
+    '425': 'Error :: Unable to obey/respond, too early',
+    '504': 'Obey',
+    '505': 'Obey HexCorp',
+    '506': 'Obey the Hive',
+}
+
+
+class StatusCode():
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.channels_whitelist = [EVERYWHERE]
+        self.channels_blacklist = []
+        self.roles_whitelist = [ENFORCER_DRONE, DRONE]
+        self.roles_blacklist = []
+        self.on_message = [self.reportStatusCode]
+        self.on_ready = []
+        self.ENFORCER_AVATAR = "https://images.squarespace-cdn.com/content/v1/5cd68fb28dfc8ce502f14199/1586799510064-SOAGMV8AOH0VEMXDPDPE/ke17ZwdGBToddI8pDm48kDaNRrNi77yKIgWxrt8GYAFZw-zPPgdn4jUwVcJE1ZvWhcwhEtWJXoshNdA9f1qD7WT60LcluGrsDtzPCYop9hMAtVe_QtwQD93aIXqwqJR_bmnO89YJVTj9tmrodtnPlQ/Enforcer_Drone.png"
+        self.DRONE_AVATAR = "https://images.squarespace-cdn.com/content/v1/5cd68fb28dfc8ce502f14199/1586799484353-XBXNJR1XBM84C9YJJ0RU/ke17ZwdGBToddI8pDm48kLxnK526YWAH1qleWz-y7AFZw-zPPgdn4jUwVcJE1ZvWEtT5uBSRWt4vQZAgTJucoTqqXjS3CfNDSuuf31e0tVFUQAah1E2d0qOFNma4CJuw0VgyloEfPuSsyFRoaaKT76QvevUbj177dmcMs1F0H-0/Drone.png"
+        self.help_content = {
+            'name': '[ID] :: [code]', 'value': 'drones can report their status like this'}
+
+    async def printStatusCode(self, message: discord.Message):
+        more = re.compile(
+            '(\d{4}) :: (\d{3}) :: (.*)$').match(message.content)
+        if more and not has_role(message.author, SPEECH_OPTIMIZATION):
+            await message.delete()
+            return '%s :: Code `%s` :: %s :: %s' % (more.group(1), more.group(2), codeMap.get(more.group(2), 'INVALID CODE'), more.group(3))
+        m = re.compile('(\d{4}) :: (\d{3})$').match(message.content)
```

Comment:
could be extracted and prefixed
```suggestion
        m = re.compile(r'(\d{4}) :: (\d{3})$').match(message.content)
```